apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -41,6 +53,26 @@ const requiredSurveyFields = ['committedToSummer', 'allStudentsShouldLearn', 'al
   'allAboutContent', 'allAboutProgramming', 'csCreativity', 'currentCsOpportunities', 'whyCsIsImportant',
   'whatTeachingSteps'];
 
+const EMAIL_RE = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+
+const isEmail = (value) => {
+  if (!value.match(EMAIL_RE)) {
+    return 'Please enter a valid email address, like name@example.com';
+  }
+};
+const isPhoneNumber = (value) => {
+  if (value.replace(/[^0-9]/g, '').length < 10) {
+    return 'Phone numbers must have at least 10 digits';
```

Comment:
Might as well give the exact phone number format (xxx) xxx-xxxx. Whether or not they do this is suspect